### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by Btpatil** [aniyomiorg/aniyomi#214] [Ideas]
Just want to give suggestions about video player. please provide the functionality in video player to pause and play on double tap and also to forward and backward 10sec. Or giving the support to play the video in external player.

**Comment by Aman9das** [comment]
Support for ext video player is present

More > setting > video i think

**Comment by DudeWithArms** [comment]
Hold anime/manga to reload library, material u, rounded corners, smooth animations. Desktop mode (for when phone is plugged into monitor) 

Of course these changes could occur over long periods of time.

**Comment by rajdeeps9957** [comment]
Suggestion about manga player.
Can you remove or add option to on/off the gap between pages in manga player.
Its okay for manga but it cuts images and makes it difficult to read manhua/webtoon.
Thank You.
